test/truefi2/Liquidator2.test.ts

```diff
@@ -65,6 +80,10 @@ describe('Liquidator2', () => {
     await tru.mint(otherWallet.address, parseEth(15e6))
     await tru.approve(stkTru.address, parseEth(1e7))
     await tru.connect(otherWallet).approve(stkTru.address, parseEth(1e7))
+
+    await creditAgency.allowPool(pool.address, true)
```

Comment:
I think we should consider adding allowing pools tusd and usdc to setupTruefi2. Maybe in a new PR.